docs/release-notes/NuGet-5.9.md

```diff
@@ -0,0 +1,215 @@
+---
+title: NuGet 5.9 Release Notes
+description: Release notes for NuGet 5.9 including new features, bug fixes, and DCRs.
+author: erdembayar
+ms.author: eryondon
+ms.date: 3/11/2021
+ms.topic: conceptual
+---
+
+# NuGet 5.9 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**5.9**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.9](https://visualstudio.microsoft.com/downloads/) | [5.0](https://dotnet.microsoft.com/download/dotnet-core/5.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2019 with .NET Core workload
+  
+> [!NOTE]
+> Visual Studio 16.9, MSBuild 16.9, and .NET 5.0.3+ requires NuGet.exe 5.9 or later.
+
+## Summary: What's New in 5.9
+
+* Add "Update" context menu item for package dependencies that launches Package Manager UI with preselected packages to update - [#10378](https://github.com/NuGet/Home/issues/10378)
+
+    ![Right-click package "Update" experience gif](media/releasenotes-59-context-update.gif)
+
+* Show the requested version (including floating version or version range request) in the "Version" column of the project list in the solution level Package Manager UI - [#9827](https://github.com/NuGet/Home/issues/9827)
+
+    ![Requested version in solution level Package Manager UI](media/releasenotes-59-requested-version.png)
+
+* IntelliCode package suggestions in the Package Manager UI Browse tab released as an A/B test - [#10053](https://github.com/NuGet/Home/issues/10053)
+
+* Extend the `.nupkg.metadata` file to include the installation source - [#10354](https://github.com/NuGet/Home/issues/10354)
+
+* Introduce a new msbuild property to exclude build output for specific TFMs during pack task - [#10396](https://github.com/NuGet/Home/issues/10396)
+
+### Issues fixed in this release
+
+**DCRs:**
+
+* INuGetPackageFileService - Fetch Images and embedded licenses for Codespaces-connected and standalone - [#10151](https://github.com/NuGet/Home/issues/10151)
```

Comment:
I would move it under bugs. It's not a DCR that needs attention.

Reply:
Will add in new commit